sources/Likutei Halachot/fill_in_english_titles.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+import csv
+import urllib2
+from get_section_titles import get_he_section_title_array
+from fuzzywuzzy import fuzz
+from fuzzywuzzy import process
+
+def match_titles(he_sections, tur_table_csv_url):
+    url = 'http://winterolympicsmedals.com/medals.csv'
+    response = urllib2.urlopen(tur_table_csv_url)
+    reader = csv.reader(response)
+    tur_table = list(reader)
+    
+    #make corresponding list to use for fuzzywuzzy
+    tur_he_title_list = []
```

Comment:
This is a great opportunity to work on list comprehensions. Can you find a way to compress lines 15 - 19 into a single line of code?